dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -112,8 +112,8 @@ dependent_startup_wait_for=rsyslogd:running
     {%- set options = options + " --skip_cmis_mgr" %}
 {% endif -%}
 
-{% if enable_xcvrd_sff_mgr %}
-    {%- set options = options + " --enable_sff_mgr" %}
+{% if enable_xcvrd_sff_mgr_controlled_tx %}
```

Comment:
@longhuan-cisco why the existing flag `enable_xcvrd_sff_mgr` will not work?

Reply:
According to https://github.com/sonic-net/sonic-platform-daemons/pull/574#discussion_r2196173037,  will keep existing flag. This PR is no longer needed.